zipline/utils/pandas_utils.py

```diff
@@ -12,6 +12,14 @@
 from distutils.version import StrictVersion
 
 pandas_version = StrictVersion(pd.__version__)
+new_pandas = pandas_version >= StrictVersion('0.19')
+skip_pipeline_new_pandas = \
+    'Pipeline categoricals are not yet compatible with pandas >=0.19'
+
+if pandas_version >= StrictVersion('0.20'):
```

Comment:
I think this function is supposed to be private. Could we just define our own version of this using the `normalize` method of timestamps? E.g.:

```
def normalize_date(dt):
    return dt.normalize()
```

Reply:
Sounds good to me.